### PR TITLE
debian7: force usage of brewed ca-certificates

### PR DESCRIPTION
**Pull request by iMichka**
- [x] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/brew/blob/HEAD/CONTRIBUTING.md) document?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/brew/pulls) for the same change?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your changes? [Here's an example](https://github.com/Homebrew/brew/blob/HEAD/Library/Homebrew/test/PATH_spec.rb).
- [ ] Have you successfully run `brew style` with your changes locally?
- [ ] Have you successfully run `brew typecheck` with your changes locally?
- [ ] Have you successfully run `brew tests` with your changes locally?

-----

### CODE REVIEW

**Comment by iMichka** [comment]
Right now this would be enabled by HOMEBREW_ON_DEBIAN7, which is a first step to fix this at least on our CI.
We might want to implement extra logic at one point for a more specific list of Linux, maybe based on the system version of curl / certificated.

**Review by Bo98 — APPROVED** [review]
Yeah, this approach makes sense to me.

File: Library/Homebrew/extend/os/mac/development_tools.rb [hunk 1]
```diff
@@ -37,13 +37,6 @@ def default_compiler
       :clang
     end
 
-    sig { returns(T::Boolean) }
-    def ca_file_handles_most_https_certificates?
```
Comment (Bo98) [inline Library/Homebrew/extend/os/mac/development_tools.rb:41]: Remove the `undef` above.

Comment (MikeMcQuaid) [reply Library/Homebrew/extend/os/mac/development_tools.rb:41]: (Line 12-13)

**Review by MikeMcQuaid — COMMENTED** [review]
Makes sense to me! Good once the `undef` is removed.

**Review by Bo98 — APPROVED** [review]

**Review by MikeMcQuaid — APPROVED** [review]
Thanks @iMichka!

**Pull request merged**